### DISCUSSION TITLE
[mini.pick] Handle case when there is only one element to select

### DISCUSSION THREAD

**Discussion by kascote** [nvim-mini/mini.nvim#1398] [General]
I was reading the help and docs and was unable to find a way to automatically make a selection when there is only one option to pick.
My use case is with `require("mini.extra").pickers.lsp` for a 'go to definition' hot hey. Most of the time there is only one reference.
TIA

**Comment by echasnovski** [comment]
Take a look at #978. Specifically [this comment](https://github.com/echasnovski/mini.nvim/issues/978#issuecomment-2178081844) for justification (with links to more details) and [this comment](https://github.com/echasnovski/mini.nvim/issues/978#issuecomment-2428497300) for inspiration about possible solution.

**Reply by kascote** [reply]
💯 makes sense. thanks for the code example!
